notifications/16102019-tutorialOffer.yml

```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
+icon: "canvas/icons/announcement.png"
+title: "Orange tutorials"
+text: "Do you want to learn how to use Orange? Check our video tutorials."
+link: "https://www.youtube.com/watch?v=HXjnDIgGDuI&list=PLmNPvQr9Tf-ZSDLwOzxpvY-HrE0yv-8Fy"
+accept_button_label: 'Tutorials'
```

Comment:
Perhaps this button should say `YouTube` to signify an external link?
Either that or just a simple `Ok`.

Reply:
Ok goes well with the text you suggested